Quarkus extension created with 2.13 stream and Quarkus CLI 3 contains Jakarta imports

### Describe the bug

I create extension with Quarkus CLI 3 for my application that uses Quarkus 2.13 and module `integration tests` of generated projects contain Jakarta imports.

### Expected behavior

Generated tests should be working for Quarkus 2.13.

### Actual behavior

```
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[19,34] package jakarta.enterprise.context does not exist
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[20,21] package jakarta.ws.rs does not exist
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[21,21] package jakarta.ws.rs does not exist
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[23,2] cannot find symbol
[ERROR]   symbol: class Path
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[24,2] cannot find symbol
[ERROR]   symbol: class ApplicationScoped
[ERROR] /home/mvavrik/Downloads/tmp/ex2/quarkus-extension-abc/integration-tests/src/main/java/io/quarkiverse/extension/abc/it/ExtensionAbcResource.java:[28,6] cannot find symbol
```

### How to Reproduce?

Steps to reproduce the behavior:

1. `quarkus create extension extension-abc --stream=2.13`
2. `cd quarkus-extension-abc/`
3. `quarkus build`

### Output of `uname -a` or `ver`

Linux apc-ap8641-3a-d11d.mgmt.pnr.lab.eng.rdu2.redhat.com 6.3.6-200.fc38.x86_64 #1 SMP PREEMPT_DYNAMIC Mon Jun  5 15:45:04 UTC 2023 x86_64 GNU/Linux 

### Output of `java -version`

openjdk version "17.0.7" 2023-04-18

### GraalVM version (if different from Java)

22.3

### Quarkus version or git rev

2.13.8.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.2 and 3.8.6

### Additional information

_No response_

--- a/devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/resteasy-codestart/java/src/main/java/org/acme/{resource.class-name}.tpl.qute.java
+++ b/devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/resteasy-codestart/java/src/main/java/org/acme/{resource.class-name}.tpl.qute.java
@@ -1,9 +1,16 @@
 package org.acme;
 
+{#if quarkus.bom.version.startsWith("2.") or quarkus.bom.version.startsWith("1.")}
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+{#else}
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.core.MediaType;
+{/if}
 
 @Path("{resource.path}")
 public class {resource.class-name} {

--- a/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/integration-tests/java/integration-tests/src/main/java/{package-name.dir}/it/{class-name-base}Resource.tpl.qute.java
+++ b/independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/integration-tests/java/integration-tests/src/main/java/{package-name.dir}/it/{class-name-base}Resource.tpl.qute.java
@@ -16,9 +16,15 @@
 */
 package {package-name}.it;
 
+{#if quarkus.bom.version.startsWith("2.") or quarkus.bom.version.startsWith("1.")}
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+{#else}
 import jakarta.enterprise.context.ApplicationScoped;
 import jakarta.ws.rs.GET;
 import jakarta.ws.rs.Path;
+{/if}
 
 @Path("/{extension.id}")
 @ApplicationScoped

